addons/website/static/src/builder/plugins/options/pricelist_option/pricelist_plugin.js

```diff
@@ -29,9 +29,16 @@ export class TogglePriceListDescriptionAction extends BuilderAction {
                 descriptionEl.classList.add(
                     params.descriptionClass,
                     "d-block",
+                    "mt-2",
                     "pe-5",
                     "text-muted"
                 );
+                if (params.descriptionExtraClass) {
+                    descriptionEl.classList.add(params.descriptionExtraClass);
+                } else if (editingElement.matches(".s_pricelist_boxed, .s_pricelist_cafe")) {
+                    // TODO: remove in master where DB will have the class thanks to the XML.
+                    descriptionEl.classList.add("o_small")
+                }
```

Comment:
You don't need this anymore, as the options templates are now static, so everyone will receive the changes 
Note that we still can't remove from these templates, as a user may have xpath them based on an attribute or something else :wink:

Reply:
@sobo-odoo done